src/components/Editor/SearchBar.js

```diff
@@ -399,12 +402,15 @@ const SearchBar = React.createClass({
   },
 
   // Handlers
-  selectResultItem(item: FormattedSymbolDeclaration) {
+  selectResultItem(e: SyntheticEvent, item: SymbolDeclaration, index?: number) {
```

Comment:
@irfanhudda thanks for the tip about line length!
The `index` was dead code. I just removed it.